<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanVerticalForm</class>
 <widget class="QWidget" name="ScanVerticalForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>145</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>140</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>155</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ScanVerticalForm{background:rgba(255,255,255,1);
border-radius:10px;
border:1px solid rgba(204,204,204,1);}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>155</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>155</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="MyToolButton" name="toolButton">
        <property name="minimumSize">
         <size>
          <width>155</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>155</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{  
color:rgb(0, 0, 0);  

}  
QToolButton:hover{
background:rgba(64,197,173,1);
color:rgb(255, 255, 255);  
 
}  
QToolButton:pressed{ 
background:rgba(64,197,173,1);
color:rgb(255, 255, 255); 
}  </string>
        </property>
        <property name="text">
         <string>招待餐费住宿类</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/imag/saomiaoreciptionnormal.png</normaloff>:/imag/saomiaoreciptionnormal.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyToolButton" name="toolButton_2">
        <property name="minimumSize">
         <size>
          <width>155</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>155</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{  
color:rgb(0, 0, 0);  

}  
QToolButton:hover{
background:rgba(64,197,173,1);
color:rgb(255, 255, 255);  
 
}  
QToolButton:pressed{ 
background:rgba(64,197,173,1);
color:rgb(255, 255, 255); 
}  </string>
        </property>
        <property name="text">
         <string>误餐费(团建)类</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/imag/saomiaoteamworknormal.png</normaloff>:/imag/saomiaoteamworknormal.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyToolButton" name="toolButton_3">
        <property name="minimumSize">
         <size>
          <width>155</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>155</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{  
color:rgb(0, 0, 0);  

}  
QToolButton:hover{
background:rgba(64,197,173,1);
color:rgb(255, 255, 255);  
 
}  
QToolButton:pressed{ 
background:rgba(64,197,173,1);
color:rgb(255, 255, 255); 
}  </string>
        </property>
        <property name="text">
         <string>无需识别验真类</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/imag/saomiaoaddworknormal.png</normaloff>:/imag/saomiaoaddworknormal.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyToolButton" name="toolButton_5">
        <property name="minimumSize">
         <size>
          <width>155</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>155</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{  
color:rgb(0, 0, 0);  

}  
QToolButton:hover{
background:rgba(64,197,173,1);
color:rgb(255, 255, 255);  
 
}  
QToolButton:pressed{ 
background:rgba(64,197,173,1);
color:rgb(255, 255, 255); 
}  </string>
        </property>
        <property name="text">
         <string>   合同类</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/imag/saomiaotravelnormal.png</normaloff>:/imag/saomiaotravelnormal.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyToolButton" name="toolButton_4">
        <property name="minimumSize">
         <size>
          <width>155</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>155</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{  
color:rgb(0, 0, 0);  

}  
QToolButton:hover{
background:rgba(64,197,173,1);
color:rgb(255, 255, 255);  
 
}  
QToolButton:pressed{ 
background:rgba(64,197,173,1);
color:rgb(255, 255, 255); 
}  </string>
        </property>
        <property name="text">
         <string>   其他类</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/imag/saomiaoothernormal.png</normaloff>:/imag/saomiaoothernormal.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyToolButton</class>
   <extends>QToolButton</extends>
   <header location="global">mytoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
